plugin/hosts/README.md

```diff
@@ -95,6 +95,10 @@ Load hosts file inlined in Corefile.
 }
 ~~~
 
+## Auto-update
```

Comment:
Could you move this into the description above as new paragraph above "This plugin can only be used once per Server Block." ?

Reply:
otherwise lgtm